MetaMask/metamask-extension
# UX: Increase network name threshold before showing tooltip

## Explanation

@rachelcope had suggested we increase the threshold for network name before showing the tooltip upon hover. I agree, since the whole Avalanche chain name fits.  A small increase makes sense.


## Manual Testing Steps

1.  Add Avalanche chain -- open network -- hover over network name -- see no tooltip
2. Add super long network name  -- open network -- hover over network name -- see tooltip upon hover

## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
